src/google-classroom-extractor/google_classroom_extractor/main.py

```diff
@@ -35,36 +42,51 @@
     USER_ACTIVITY_ROOT_DIRECTORY,
 )
 
+logger: logging.Logger
 
-def request() -> Result:
-    load_dotenv()
+
+def configure_logging():
+    global logger
+
+    logger = logging.getLogger(__name__)
+
+    level = os.environ.get("LOGLEVEL", "INFO")
     logging.basicConfig(
         handlers=[
             logging.StreamHandler(sys.stdout),
         ],
-        format="%(asctime)s - %(levelname)s - %(message)s",
-        level=os.environ.get("LOGLEVEL", "INFO"),
+        format="%(asctime)s - %(levelname)s - %(name)s - %(message)s",
+        level=level,
     )
-    logging.getLogger("matplotlib").setLevel(logging.ERROR)
-    logging.getLogger("opnieuw").setLevel(logging.DEBUG)
```

Comment:
@bradbanister do you agree that we don't need matplotlib here at all, and that opnieuw should log at the level that we just set in the `basicConfig(...)` call?

Reply:
yeah, that's fine 👍